packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -62,18 +67,23 @@ export const publishInterchainAssetFromBank = async (
   },
   { options: { interchainAssetOptions } },
 ) => {
-  const { denom, decimalPlaces, proposedName, keyword } =
-    interchainAssetOptions;
+  const {
+    denom,
+    decimalPlaces,
+    keyword,
+    issuerName = keyword,
+    proposedName = keyword,
+  } = interchainAssetOptions;
 
   // Incompatible with issuerBoardId.
   assert.equal(interchainAssetOptions.issuerBoardId, undefined);
   assert.typeof(denom, 'string');
-  assert.typeof(keyword, 'string');
-  assert.typeof(decimalPlaces, 'number');
+  assert.typeof(issuerName, 'string');
   assert.typeof(proposedName, 'string');
+  assert.typeof(decimalPlaces, 'number');
```

Comment:
moving this line looks superfluous to me. If you are going to move it, consider some rational ordering like alphabetical (as you have in the destructuring for this fn